ci-operator/step-registry/openshift/e2e/azure/resourcegroup/openshift-e2e-azure-resourcegroup-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: openshift-e2e-azure-resourcegroup
+  steps:
+    pre:
+    - chain: ipi-azure-pre-resourcegroup
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: ipi-azure-post
```

Comment:
Do you want to add a post step for deleting the resource group in case the installation failed before creating the metadata.json?